plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -680,7 +678,7 @@ public void setupClass()
         executor = newCachedThreadPool(daemonThreadsNamed("hive-%s"));
         heartbeatService = newScheduledThreadPool(1);
         // Use separate staging directory for each test class to prevent intermittent failures coming from test parallelism
-        temporaryStagingDirectory = createTempDirectory("trino-staging-");
+        temporaryStagingDirectory = createTempDirectory("trino-staging-${USER}-");
```

Comment:
this creates a path with literal `${USER}`:
```
/var/folders/dr/jhnfvsrd2zl925dyfsctj0pr0000gn/T/trino-staging-${USER}-7987395433279077839
```

createTempDirectory is randomized anyway, so revert

Reply:
if we want this to be a path template (to be replaced later), let's not call `createTempDirectory` on this / or create directory created here, as it's not going to be used

```
// createTempDirectory is used only to generate path to the temporary folder
temporaryStagingDirectory = createTempDirectory("trino-staging-${USER}-");
Files.delete(temporaryStagingDirectory);
```